install/migrations/update_10.0.x_to_10.1.0/dcroom.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2022 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ */
+
+if (!$DB->fieldExists("glpi_dcrooms", "vis_cell_width")) {
+    $migration->addField(
+        "glpi_dcrooms",
+        "vis_cell_width",
+        "int",
+        [
+            'after'   => "vis_rows",
+            'value'   => 40,
+        ]
+    );
+}
+if (!$DB->fieldExists("glpi_dcrooms", "vis_cell_height")) {
+    $migration->addField(
+        "glpi_dcrooms",
+        "vis_cell_height",
+        "int",
+        [
+            'after'   => "vis_cell_width",
+            'value'   => 39,
```

Comment:
Maybe a new default to `40` would be less disturbing. Using `update` would permit to keep previous size on existing items.
```suggestion
            'value'   => 40,
            'update'  => 39,
```